styles/style.css

```diff
@@ -24,6 +24,28 @@ button {
   font-size: 1rem;
 }
 
+button {
+  padding: 0.4rem 0.6rem;
+  background-color: var(--orange-dark);
+  border: none;
+  color: var(--white);
+  border-radius: 0.2rem;
+  font-weight: 600;
+}
+button:hover {
+  background-color: rgb(110, 108, 218);
+}
+
+.volunteer-btn {
```

Comment:
Well done on creating this variation on the button! 

Something you could try next time is to use a CSS naming convention for creating these kind of modifier classes. You use the name of the root class, followed by `--`, followed by the name of the modifier, e.g. in your case
```
.button--volunteer {
    background-color: var(--white);
    color: var(--orange-dark);
}
```

Then imagine in the future if you wanted to add another class for a purple donate button, you could create
```
.button--donate {
    background-color: var(--purple);
    color: var(--orange-dark);
}
```
This is just a naming convention, but semantically we can assume that your classes are used on button elements and will share the styles applied to button :)